avocado/core/test.py

```diff
@@ -52,17 +52,6 @@
 COMMON_TMPDIR_NAME = 'AVOCADO_TESTS_COMMON_TMPDIR'
 
 
-class NameNotTestNameError(Exception):
-
-    """
-    The given test name is not a TestName instance
-
-    With the introduction of :class:`avocado.core.test.TestName`, it's
-    not allowed to use other types as the ``name`` parameter to a test
-    instance.  This exception is raised when this is attempted.
-    """
-
-
```

Comment:
Up to here I can not tell why this is needed, but my guess is that I will discover soon, in the next commits. Maybe this change should come together with the code that actually uses it? Or, if that's not the case, a better context in the commit message would suffice.